packages/@n8n/eslint-config/src/rules/no-import-enterprise-edition.ts

```diff
@@ -0,0 +1,41 @@
+import { ESLintUtils } from '@typescript-eslint/utils';
+
+export const NoImportEnterpriseEditionRule = ESLintUtils.RuleCreator.withoutDocs({
+	meta: {
+		type: 'problem',
+		docs: {
+			description:
+				'Disallow imports from .ee directories in non-enteprise code. Only code in .ee directories can import from other .ee directories.',
```

Comment:
Typo in error message:
```suggestion
				'Disallow imports from .ee directories in non-enterprise code. Only code in .ee directories can import from other .ee directories.',
```